src/vs/workbench/contrib/terminal/browser/terminalInstance.ts

```diff
@@ -684,12 +684,6 @@ export class TerminalInstance extends Disposable implements ITerminalInstance {
 
 		xterm.open(this._xtermElement);
 
-		const suggestedRendererType = this._storageService.get(SUGGESTED_RENDERER_TYPE, StorageScope.GLOBAL);
-		if (this._configHelper.config.gpuAcceleration === 'auto' && (suggestedRendererType === 'auto' || suggestedRendererType === undefined)
-			|| this._configHelper.config.gpuAcceleration === 'on') {
-			this._enableWebglRenderer();
-		}
-
```

Comment:
I deleted this as `this._enableWebglRenderer()` is called again inside `this.updateConfig();` line 819